.pre-commit-config.yaml

```diff
@@ -25,9 +25,10 @@ repos:
         exclude: ^docs/conf.py
         additional_dependencies:
           - types-dataclasses >= 0.1.3
-          - types-toml >= 0.1.1
+          - tomli >= 0.2.6
```

Comment:
Do you want an upper bound here too?

Reply:
:+1: changed it for consistency now that you mentioned. Though it's just for mypy so shouldn't matter too much :smile_cat: